src/notes.py

```diff
@@ -0,0 +1,53 @@
+import parselmouth
+
+import sys
+import math
+import os
+
+
+def frequency_to_note(frequency):
+    '''
+    A4 440 Hz
+    '''
+    if frequency == 0:
+        return None
+    return round(12*math.log2(frequency/440.0)+45) % 12
+
+
+def note_to_symbol(note):
+    if note == None:
+        return 'None'
+    symbols = ['C', 'C#', 'D', 'D#', 'E', 'F', 'F#', 'G', 'G#', 'A', 'A#', 'H']
+    return symbols[note]
+
+
+class NotesInTime:
+    def __init__(self, file):
+        snd = parselmouth.Sound(file)
+        pitch = snd.to_pitch()
+        frequencies = pitch.selected_array['frequency']
+        notes = [frequency_to_note(freq) for freq in frequencies]
+        self.notesInTime = []
```

Comment:
It should be first line of `__init__` so that we can see all class fields without examining whole `__init__` method

Reply:
Done